bin/p2-preparer/main.go

```diff
@@ -34,6 +34,12 @@ func main() {
 		logger.WithError(err).Fatalln("invalid parameter")
 	}
 
+	if preparerConfig.ReadyFile != "" {
+		if _, err := os.Stat(preparerConfig.ReadyFile); os.IsNotExist(err) {
+			logger.WithError(err).Fatalln("ready file not present")
```

Comment:
Is the intent of this change to make the preparer fail to operate if the ready file is not present? As an aside, we try to give thorough commit messages explaining the intent and motivation for each change so questions like this are easy to answer when looking at old code